enterprise/frontend/src/metabase-enterprise/clean_up/CleanupCollectionModal/CleanupCollectionModalFilters.tsx

```diff
@@ -0,0 +1,53 @@
+import { c, t } from "ttag";
+import _ from "underscore";
+
+import { Icon, Select, Flex, Text, Switch } from "metabase/ui";
+
+import { dateFilterOptions, isDateFilter, type DateFilter } from "./utils";
+
+interface CleanupCollectionModalFiltersProps {
+  dateFilter: DateFilter;
+  recursiveFilter: boolean;
+  onDateFilterChange: (dateFilter: DateFilter) => void;
+  onRecursiveFilterChange: (recursiveFilter: boolean) => void;
+}
+
+export const CleanupCollectionModalFilters = ({
+  dateFilter,
+  recursiveFilter,
+  onDateFilterChange,
+  onRecursiveFilterChange,
+}: CleanupCollectionModalFiltersProps) => (
+  <Flex
+    mt="1.5rem"
+    mb="1rem"
+    w="100%"
+    justify="space-between"
+    direction={{ base: "column", md: "row" }}
+    align={{ base: "start", md: "center" }}
+    gap={{ base: "md", md: "none" }}
+    style={{ marginInlineEnd: ".25rem" }}
+  >
+    <Text fw="bold" display="inline-flex" style={{ alignItems: "center" }}>
+      {c(`Not used in over <time-span>`).jt`Not used in over ${(
```

Comment:
This isn't a bad way to provide context, but for the sake of clarity it might be worth sticking with our existing pattern which is to explain the meaning of the placeholder, with something like "{0} is a duration". Not saying this way of explaining ourselves is inherently better than the approach you've taken - just that consistency will help the translators out.

I wouldn't be totally opposed to changing all of our context strings over to using this kind of pattern (perhaps without the hyphen) fwiw

Reply:
You might find that you get a console warning since the component inside `jt` doesn't have a `key` prop set